### PR TITLE
Get tile_config addr from python

### PR DESCRIPTION
**Pull request by MarisaSparkL**
### PR Category
CINN

### PR Types
Others

### Description
Pcard-74042
* Copy tile_config directory to paddle install directory in setup.py.
* In paddle __init__.py, use pkg_resources to get the install path of paddle.base and write it to 'CINN_CONFIG_PATH' in env.
*  In file_database.cc, read 'CINN_CONFIG_PATH' from env to get the default path of tile_config when FLAGS_cinn_tile_config_filename_label set to default.
* Use test_path in file_tile_config_test ,tile_config_performance_test and tile_config_searcher_test.
* Copy cinn config json files to python/paddle/cinnconfigs path

### CODE REVIEW

File: paddle/cinn/ir/group_schedule/config/file_database.cc [hunk 1]
```diff
@@ -106,14 +106,24 @@ std::string IterSpaceTypeToDir(const common::Target target,
                             test_path));
     }
   };
+  const char* envValue = getenv("CINN_CONFIG_PATH");
```
Comment (BiynXu) [inline paddle/cinn/ir/group_schedule/config/file_database.cc:109]: const char* config_path_env_value

File: python/paddle/__init__.py [hunk 2]
```diff
@@ -589,6 +589,11 @@
     if os.path.exists(cuh_file):
         os.environ.setdefault('runtime_include_dir', runtime_include_dir)
 
+    import pkg_resources
+
+    data_file_path = pkg_resources.resource_filename('paddle.cinnconfig', '')
```
Comment (BiynXu) [inline python/paddle/__init__.py]: cinnconfig -> cinn_config

File: python/paddle/cinnconfig/__init__.py
Comment (BiynXu) [inline python/paddle/cinnconfig/__init__.py:1]: cinnconfig -> cinn_config

File: python/paddle/cinnconfig/tile_config/NVGPU_Tesla_V100_SXM2_32GB/R_EREBE/Rdynamic.json
Comment (BiynXu) [inline python/paddle/cinnconfig/tile_config/NVGPU_Tesla_V100_SXM2_32GB/R_EREBE/Rdynamic.json:1]: empty file先删掉

File: setup.py [hunk 3]
```diff
@@ -1040,6 +1040,23 @@ def get_paddle_extra_install_requirements():
         return []
 
 
+def get_cinnconfig_jsons():
```
Comment (BiynXu) [inline setup.py]: get_cinnconfig_jsons -> get_cinn_config_jsons

File: setup.py [hunk 4]
```diff
@@ -1055,6 +1072,20 @@ def get_package_data_and_package_dir():
     package_data['paddle.base'] += [
         paddle_binary_dir + '/python/paddle/cost_model/static_op_benchmark.json'
     ]
+
+    whl_cinnconfig_path = paddle_binary_dir + '/python/paddle/cinnconfig'
```
Comment (BiynXu) [inline setup.py]: whl_cinnconfig_path -> whl_cinn_config_path

File: setup.py [hunk 5]
```diff
@@ -1055,6 +1072,20 @@ def get_package_data_and_package_dir():
...
+    src_cinnconfig_path = (
```
Comment (BiynXu) [inline setup.py]: src_cinnconfig_path -> src_cinn_config_path

File: python/paddle/__init__.py (hunk 2 above)
Comment (MarisaSparkL) [reply python/paddle/__init__.py]: Done

File: python/paddle/cinnconfig/__init__.py
Comment (MarisaSparkL) [reply python/paddle/cinnconfig/__init__.py:1]: Done

File: python/paddle/cinnconfig/tile_config/NVGPU_Tesla_V100_SXM2_32GB/R_EREBE/Rdynamic.json
Comment (MarisaSparkL) [reply python/paddle/cinnconfig/tile_config/NVGPU_Tesla_V100_SXM2_32GB/R_EREBE/Rdynamic.json:1]: Done

File: setup.py (hunk 3 above)
Comment (MarisaSparkL) [reply setup.py]: Done

File: setup.py (hunk 4 above)
Comment (MarisaSparkL) [reply setup.py]: Done.

File: setup.py (hunk 5 above)
Comment (MarisaSparkL) [reply setup.py]: Done.

**Review by BiynXu — APPROVED** [review]

**Pull request merged**